changelog/unreleased/issue-323

```diff
@@ -9,5 +9,10 @@ and destination repository. Also, the transferred files are not re-chunked,
 which may break deduplication between files already stored in the
 destination repo and files copied there using this command.
 
+The init command now supports a `--copy-chunker-parameters` parameter to
+setup repositories with identical parameters for splitting files into chunks.
+This allows deduplication to work between snapshots from such repositories.
```

Comment:
What do you think about changing this paragraph to the following, to make it more of a follow-up to the previous paragraph?

~~~
To fully support deduplication between repositories when the copy command is
used, the init command now supports the `--copy-chunker-parameters` option,
which initializes the new repository with identical parameters for splitting files
into chunks as an already existing repository. This allows copied snapshots to be
equally deduplicated in both repositories.
~~~

Reply:
I've changed the paragraph. Thanks for the suggestion.